src/pkg/services/m365/api/events_pager_test.go

```diff
@@ -107,13 +108,10 @@ func (suite *EventsPagerUnitSuite) TestEventsList() {
 			Reply(http.StatusOK).
 			JSON(validEventsListSingleNextLinkResponse)
 
-		// Number of retries and delay between retries is handled by a kiota
-		// middleware. We can change the default config parameters when setting
-		// up the mock in a later PR.
 		gock.New(graphAPIHostURL).
 			Get(nextLinkPath).
 			AddMatcher(pageSizeMatcher(t, expectedPageSize)).
-			Times(4).
+			Times(2). // retry count is configured to 1
```

Comment:
❤️ thank you for getting around to this before I happened to. It was on my list to cleanup